src/modules/Ticket/index.tsx

```diff
@@ -100,12 +100,12 @@ function Ticket() {
   const ValidationSchema = yup.object({
     title: yup
       .string()
-      .matches(/^[-()., A-Za-z0-9\n]*$/i, 'Special characters are not allowed.')
+      .matches(/^[-()., A-Za-z0-9\n/{}[\]]*$/i, 'Special characters are not allowed.')
       .required('Complaint/Request title is required'),
     description: yup
       .string()
       .matches(
-        /^[-().,_ A-Za-z0-9@': \n]*$/i,
+        /^[-().,_ A-Za-z0-9@': \n/{}[\]]*$/i,
```

Comment:
Should we also escape `}` and `[` in the regex?

Reply:
Yes..its escaping } and [ without giving extra escape character (\)